R/net.dat.R

```diff
@@ -162,6 +167,28 @@ set_attr <- function(dat, item, value) {
   return(dat)
 }
 
+#' @param n_new the number of new elements to append at the end of `item`
+#' @rdname dat_get_set
+#' @export
+append_attr <- function(dat, item, value, n_new) {
+  if (!item %in% names(dat[["attr"]])) {
```

Comment:
also, I prefer to keep function arguments with period case, so n.new instead of n_new

Reply:
ok